Root/SusyNtMaker.cxx

```diff
@@ -1522,6 +1547,25 @@ struct FillCutFlow { ///< local function object to fill the cutflow histograms
     FillCutFlow& enableFilterNextCuts() { includeThisCut_ = true; return *this; }
 };
 //----------------------------------------------------------
+void SusyNtMaker::fillTriggerHisto() // dantrim trig
+{
+    for (int i = 0; i < triggerbits::N_TRIG; i++) {
+        if(m_trigTool->isPassed(triggerbits::trigger_names[i])) h_passTrigLevel->Fill(i+0.5);
```

Comment:
here can you use `m_evtTriggerBits.TestBitNumber()` instead?
(avoid calling `isPassed` multiple times)